app/models/taxon_parent_links_update.rb

```diff
@@ -0,0 +1,11 @@
+class TaxonParentLinksUpdate
```

Comment:
What I was trying to say is that I'm a bit worried about the growth of content-tagger as a whole. It looks like clearing of the parent links could be reduced to a couple of lines in the rake task. Extracting some parts it into `ClearParentLink` and `TaxonParentLinksUpdate` seems to introduce a lot of indirection. Especially because those classes are single-use and will never be called again after running it now. WDYT?